package.json

```diff
@@ -33,7 +33,8 @@
     "sinon": "^15"
   },
   "exports": {
-    ".": "./src/browser/index.js"
+    ".": "./src/browser/index.js",
+    "./pause.js": "./src/server/pause.js"
```

Comment:
It's unfortunate that we need to export this even though consumers don't need to know it exists. Wondering if we could just inject it into the HTML document that `wtr-config.js` generates, similar to the ResizeObserver error handler and visual-diff styles.